### ISSUE TITLE
Inra-module navigation

### ISSUE THREAD

**Issue by nisargjhaveri** [nisargjhaveri/saathi#25]
Something like go to module's main page..

**Comment by nisargjhaveri** [comment]
@imjalpreet, Take a look at this when free.

**Comment by gauravmittal1995** [comment]
@nisargjhaveri Do we need a separate page with links to modules?? For easy accessibility, we should just have a side bar as it is currently. I mean, when people r using it, the priority will be for fast navigation. 
This is my opinion. We can discuss this though.

**Comment by nisargjhaveri** [comment]
Oh! I think you misinterpreted it. What I meant was, when you are at `missing/report` you should have some way to go to `missing` other then the sidebar, as through side-bar it is 2-clicks away and you have to search through all modules again for `missing`.

It would be nice if have a 'go to parent module' link in each page inside a module.

**Comment by gauravmittal1995** [comment]
Oh, Sorry! Totally misunderstood that!! I agree to the above :+1: +1

**Comment by gauravmittal1995** [comment]
We should have this for easy accessibility.
